src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -1982,6 +1973,35 @@ public void validateDeleteRequest(
     return values;
   }
 
+  private static final String snapshotDataTemplate =
```

Comment:
Our code hasn't followed java conventions here before but I think it would be good to migrate our code to follow them as we go. Following the conventions, this should be all uppercase.
```suggestion
  private static final String SNAPSHOT_DATA_TEMPLATE =
```